app/views/shared/_footer_lite.html.slim

```diff
@@ -3,9 +3,9 @@ footer.footer.bg-navy.white
     .clearfix
       .sm-col = t('shared.footer_lite.gsa')
       .sm-col-right.caps
-        = link_to t('links.help'), help_path,
+        = link_to t('links.help'), "https://#{APP_NAME}/help",
```

Comment:
Ok that makes sense! Since it turns out there are other links to this scattered throughout the app, I pulled everything into a `MarketingService` object in 68cf62b2aa140b845f2da201c7dc1cf5fda4cedb, PTAL!